dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/osgi/WebContainer.java

```diff
@@ -223,7 +226,13 @@ protected void unsetWebMBeanRuntime(ServiceReference<WebMBeanRuntime> ref) {
     private static final int DEFAULT_MAX_VERSION = 30;
     private ServiceReference<ServletVersion> versionRef;
     
-    private static boolean serverStopping = false;
+    private static StaticValue<AtomicBoolean> serverStopping = StaticValue.createStaticValue(new Callable<AtomicBoolean>() {
```

Comment:
Again, doesn't have to be done here, but this would be a good one to get rid of.. WebApp could be a ServerQuiesceListener rather than calling a static method on WebContainer.

Reply:
Yes, this one probably could be replaced easily with non-static.  At this point I wanted to simply identify all the 'bad' statics.  At first I was going to try and get rid of them, but that quickly turned into a massive exercise.  Now we can easily search for all callers of StaticValue.createStaticValue to find some technical dept to clean up.